spec/mutations/sequences/create_spec.rb

```diff
@@ -0,0 +1,26 @@
+require 'spec_helper'
+
+describe Steps::Create do
+  let(:user) { FactoryGirl.create(:user) }
+  let(:mutation) { Sequences::Create }
+  let(:step) { {message_type: 'move_rel',
```

Comment:
Avoid using {...} for multi-line blocks.<br>Block body expression is on the same line as the block start.<br>Space inside { missing.<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.